### PR TITLE
Filter content

### PR DESCRIPTION
**Pull request by SvenDub**
Filter the content by positivity.

---

Before submitting your pull request, consider the following: Does your code...

- [x] Implement functionality
- [x] Contain comments/documentation
- [x] Build without errors
- [x] Contain and pass unit testing
- [x] Conform to the Style Guide
- [x] Have no merge conflicts with ```master```

When you've checked all of the boxes above your code is up for review. If approved it can be merged with the ```master``` branch.

If you want feedback on your work before having finished your code, add ```[WIP]``` to the title of your PR.

### CODE REVIEW

File: crawler/src/main/java/nl/fhict/happynews/crawler/analyzer/PositivityAnalyzer.java [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+ */
+@Component
+public class PositivityAnalyzer {
+
+    @Autowired
+    private ApplicationContext applicationContext;
+
+    private Logger logger = LoggerFactory.getLogger(PositivityAnalyzer.class);
+
+    private HashSet<String> positiveWords;
```
Comment (TvanBronswijk) [inline crawler/src/main/java/nl/fhict/happynews/crawler/analyzer/PositivityAnalyzer.java]: Could just be a `Set`.

File: crawler/src/main/java/nl/fhict/happynews/crawler/analyzer/PositivityAnalyzer.java [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+            if (positiveWords.contains(word)) {
+                pos.addAndGet(count);
+            } else if (negativeWords.contains(word)) {
+                neg.addAndGet(count);
+            }
+        });
+
+        int positive = pos.get();
+        int negative = neg.get();
+        boolean isPositive = 0.7203 * positive - negative > 3;
```
Comment (TvanBronswijk) [inline crawler/src/main/java/nl/fhict/happynews/crawler/analyzer/PositivityAnalyzer.java]: Add a comment that explains this formula

File: crawler/src/main/java/nl/fhict/happynews/crawler/analyzer/PositivityAnalyzer.java [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+
+    /**
+     * Gets the unique words per text and their frequency.
+     *
+     * @param inputText
+     * @return
+     */
+    private HashMap<String, Integer> getUniqueWords(String inputText) {
+        String[] words = inputText.split("[,.:;\\s\\n]");
+        HashMap<String, Integer> uniqueWords = new HashMap<>();
```
Comment (TvanBronswijk) [inline crawler/src/main/java/nl/fhict/happynews/crawler/analyzer/PositivityAnalyzer.java]: Should be a `Map`.

File: crawler/src/main/java/nl/fhict/happynews/crawler/analyzer/PositivityAnalyzer.java [hunk 4]
```diff
@@ -0,0 +1,109 @@
...
+    }
+
+    /**
+     * Load word from a file
+     *
+     * @param file filename for file located in resource folder
+     * @return set of words
+     */
+    private HashSet<String> loadWords(String file) {
+        HashSet<String> words = new HashSet<>();
```
Comment (TvanBronswijk) [inline crawler/src/main/java/nl/fhict/happynews/crawler/analyzer/PositivityAnalyzer.java]: Should be a `Set`.

File: crawler/src/main/java/nl/fhict/happynews/crawler/analyzer/PositivityAnalyzer.java (hunk 2 above)
Comment (SanderVanAndel) [inline crawler/src/main/java/nl/fhict/happynews/crawler/analyzer/PositivityAnalyzer.java]: And make the 0.7203 a static final variable if it's a constant value

**Pull request merged**